fix: allow Respect to send a null payload (#2548)
Redocly/redocly-cli: .changeset/wet-months-wonder.md

---
"@redocly/respect-core": patch
---

> Allow Respect to send a null payload when an example is described in the OpenAPI schema

Reviewer: ```suggestion
Respect can now send a null payload when an example is described in the OpenAPI schema.
```
